lib/SIL/IR/SILPrinter.cpp

```diff
@@ -3433,6 +3433,36 @@ static void printSILDifferentiabilityWitnesses(
     dw->print(Ctx.OS(), Ctx.printVerbose());
 }
 
+static void printSILLinearMapTypes(SILPrintContext &Ctx,
```

Comment:
Is this only for printing branching trace enums now?

Reply:
Yes